packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -121,6 +121,10 @@ class Config:
         }
 
 
+class NodeOutputs(InputSchemaWithoutCameCase):
```

Comment:
```suggestion
class NodeOutputs(InputSchemaWithoutCamelCase):
```

Reply:
subtle :-D took me a while to see the missing tail of the camel :-)

Reply:
Good point. Thanks. Fixed